deploy/helm/omni/values.yaml

```diff
@@ -0,0 +1,72 @@
+global: {}
+
+omni:
```

Comment:
I don't see the need for the nesting. This is the Omni chart so all values should just be at the "base" i.e. `.Values.debug` etc...

Reply:
The consideration here was that we might want to include other components down the line, such as image factory or discovery service. Nesting by `omni` would allow the variables for those apps to be included in the same valuesfile.

Reply:
I would have them as separate charts and pull them in as chart dependencies. This automatically creates a "nested" Values field for the external charts.

Reply:
As discussed, I removed the nesting